src/loaders/common/loader_common_names.py

```diff
@@ -16,12 +16,17 @@
 KBASE_COLLECTION_ARG_NAME = "kbase_collection"
 # Description of the --kbase_collection argument in various loaders programs.
 KBASE_COLLECTION_DESCR = (
-    f"KBase collection identifier name (default: {DEFAULT_KBASE_COLL_NAME})."
```

Comment:
Missed one thing - is this going to affect other CLIs?

Reply:
I think we are moving away from having `GTDB` or any others as default collection. Currently, only the  taxa_count and gtdb_genome_attribs_load has default kbase_collection as `GTDB`. I think for a description, that's fine.

Reply:
:+1: